src/components/ImportFile/ImportFileContent/ImportFileAccept.js

```diff
@@ -0,0 +1,58 @@
+import ClassNames from 'classnames';
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Icon } from '../../Icon';
+import { ProgressBar } from '../../ProgressBar';
+
+const ImportFileAccept = ({
+  progressDone,
+  acceptedFiles,
+  rejectedFiles,
+  text,
+  icon,
+  className,
+  ...props
```

Comment:
no need for this destructure.  it does not look used below.

Reply:
Not sure I understand, i'm using all of these props, giving more options for the consumer

Reply:
Sorry, I was speaking more specifically to the `...props` spread would have been a better word.  You can leave that off since the object named `props` is not being used.  I like the props you are using and definitely don't want any of them removed.